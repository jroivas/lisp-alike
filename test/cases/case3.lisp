(def! mylist (fn* (& items) items))
(def! a (list 1 2 3))
(def! b (mylist 1 2 3))
(= a b)
