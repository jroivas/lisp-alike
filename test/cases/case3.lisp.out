#function: (fn* (& items) items)
(1 2 3)
(1 2 3)
true
